lib/core/include/irods/authentication_plugin_framework.hpp

```diff
@@ -28,6 +28,7 @@ namespace irods::experimental::auth
 {
     static const char* const flow_complete{"authentication_flow_complete"};
     static const char* const next_operation{"next_operation"};
+    static const char* const force_password_prompt{"force_password_prompt"};
```

Comment:
This change should be retained regardless of the decision about whether to keep this commit or not.

Reply:
After discussion, we have decided that the plugin will no longer support the password key. The latest commit removes this change and retains the good stuff.